src/Form.php

```diff
@@ -740,7 +740,7 @@ public function filterValue(mixed $field): mixed
         }
 
         if (($field instanceof AbstractElement) && !($field instanceof Checkbox) &&
-            !($field instanceof Radio) && ($realValue !== null) && ($realValue != '')) {
```

Comment:
I'm not sure what the reasoning behind this decision is, but I would like to be able to filter values to an empty `string` or `null`.